### PR TITLE
Add  and remove

### PR DESCRIPTION
**Pull request by techmoves**
In this project this is what i did 👍 

- [x] Remove all hardcoded items from the tasks array.
- [x] No linters error

- [x] Create a new JavaScript file for the new functionality.

- [x] Implement a function for adding a new task (add a new element to the array).

- [x] Implement a function for deleting a task (remove an element from the array).

- [x] Implement a function for editing task descriptions.
- [x] all changes are saved on local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
+    const input = form.text.value;
+    const completed = false;
+    const id = storeData.length;
+
+    script.add(input, completed, id);
+    display();
+    form.text.value = '';
+  });
+};
+renderList();
```
Comment (OyePriscilla) [inline src/index.js:56]: - [ ] Please, I observed that if I refresh the browser after performing operation on your app, the `todos` in the `localstorage` were not displayed to the UI. Kindly fix this. 👍 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @techmoves 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Removed all hardcoded items from the tasks array ✔️
Implement a function for adding, editing and deleting a task ✔️
All changes to the To Do List saved in local storage ✔️

## Required Changes ♻️

- [x] Kindly ensure that all these links in your README file redirect visitors to your repo to the appropriate page (1. Client section "**linters**", 2. **license section**, 3. Contirbution section "**issues page**" and 4. deployment section "**GitHub pages**" )

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (techmoves) [reply src/index.js:56]: hi [OyePriscilla](https://github.com/OyePriscilla)

when you add a new todos they show-up on the UI

Comment (OyePriscilla) [reply src/index.js]: @techmoves Yes, they do.

File: README.md [hunk 2]
```diff
@@ -123,6 +123,14 @@ To run tests, run the following command:
 
 - Project is deployed using [GitHub Pages](https://github.com/techmoves/todo-list.git)
```
Comment (addod19) [inline README.md:124]: - [ ] As mentioned in the previous review, the link redirecting users to GitHub pages is misleading, instead it leads to the GitHub repo

File: modules/script.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const save = (data) => {
+  localStorage.setItem('mylist', JSON.stringify(data));
+};
+
+const retrieve = () => JSON.parse(localStorage.getItem('mylist'));
```
Comment (addod19) [inline modules/script.js:5]: - [ ] As mentioned in the previous code review, data does not persist after browser page refresh. Adding and removing data shows the data displayed in the UI, however, when we decide to refresh the page, all data stored in the UI disappears although we can see all the data stored in the localstorage

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @techmoves, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Nice job fixing some of the comments from the previous code review :rocket: 


 ## Required Changes ♻️
_Check the comments under the review._
- [x] Here is a screenshot of the said bug
Screenshot from 2023-05-12 14-21-57


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @techmoves 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Data in the `localStorage` displayed correctly in the UI ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**